### PR TITLE
Homeworke14-completed

### PR DESCRIPTION
**Pull request by MAXPAIN007**
Homeworke14-completed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,6 @@ lesson13/__init__.py
 lesson13/json__ponomarov.log
 lesson14/__init__.py
```
Comment (DartBrovsky) [inline .gitignore:9]: __init__ files very necessary to be in your PR. It is very important for correct work of your python packages. Please delete this PR and push new one without __init__ files in .ginignore file.

File: lesson16/part1/employee.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+
+class Employee:
+
+    name:str
```
Comment (DartBrovsky) [inline lesson16/part1/employee.py:5]: It is better to create class fields private or at least protected if we do not want to provide an access from outside.

**Review by DartBrovsky — CHANGES_REQUESTED** [review]
Reviewed. Please take a look into my comments. Thanks.

File: .gitignore (hunk 1 above)
Comment (MAXPAIN007) [reply .gitignore:9]: Before this, you were asked to remove all unnecessary files including __init __. Try to decide.

**Pull request closed without merging**